layers/+misc/multiple-cursors/config.el

```diff
@@ -12,4 +12,5 @@
 ;;; License: GPLv3
 
 (defvar multiple-cursors-backend 'evil-mc
-  "The package used as the backend for multiple cursors functionality.")
+  "The package used as the backend for multiple cursors functionality.
+Possible values are `mc' or `evil-mc'.")
```

Comment:
Not relevant to this PR, probably added by mistake.

Reply:
I added it while at it, I agree it is not related to this PR and adding a bit of noise but I didn't feel like opening a new PR just for that. I'll remove it if you prefer.

Reply:
Ok, in that case leave it here, no need for a new PR.